src/navigation/application-launcher.component.js

```diff
@@ -77,8 +77,9 @@
    </div>
  </file>
  <file name="script.js">
-   angular.module('patternfly.navigation').controller('applicationLauncherController', ['$scope',
-     function ($scope) {
+   angular.module('patternfly.navigation').controller('applicationLauncherController', ['$scope', '$window',
+     function ($scope, $window) {
+       $scope.imagePath = $window.IMAGE_PATH || "img";
```

Comment:
and again...

Reply:
...again, I was preserving what was previously present.  Happy to converge on a single path with this PR though.